modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java

```diff
@@ -492,10 +492,8 @@ private static Stream<DefaultValueArg> defaultValueArgs() {
                 // new DefaultValueArg("TIMESTAMP WITH LOCAL TIME ZONE", "TIMESTAMP '2021-01-01 01:01:01'"
                 //         , LocalDateTime.parse("2021-01-01T01:01:01")),
 
-                new DefaultValueArg("BINARY(3)", "x'010203'", new byte[]{1, 2, 3})
-
-                // TODO: IGNITE-17374
```

Comment:
seems we need to close this issue ?

Reply:
All issues from the patch mentioned in JIRA description and will be closed after merge will be done